# HG changeset patch
# User Rik <rik@octave.org>
# Date 1362452667 28800
#      Mon Mar 04 19:04:27 2013 -0800
# Node ID 741dbca67d80c7640c91fa6c031486dc2c5972ca
# Parent  049c769b11c56287ae7396db2d69941e3a0ca42b
find-docstring-files.sh: Fix typo in comment.

* doc/interpreter/find-docstring-files.sh: Fix typo in comment.

diff --git a/doc/interpreter/find-docstring-files.sh b/doc/interpreter/find-docstring-files.sh
--- a/doc/interpreter/find-docstring-files.sh
+++ b/doc/interpreter/find-docstring-files.sh
@@ -1,16 +1,16 @@
 #! /bin/sh
 
 if [ $# -ne 1 ]; then
   echo "usage: find-docstring-files TOP-SRCDIR" 1>&2
   exit 1
 fi
 
-## if there is a file in teh build directory tree, assume it is
+## if there is a file in the build directory tree, assume it is
 ## the file we are looking for.  Otherwise, get the one from the
 ## source tree.
 
 if [ -f "../../scripts/DOCSTRINGS" ]; then
   echo "../../scripts/DOCSTRINGS"
 else
   echo "$1/scripts/DOCSTRINGS"
 fi
